msk/actions/create.py

```diff
@@ -226,6 +283,11 @@ def name(self) -> str:
         ),
         intent_name=s.intent_name
     ))
+    if Lazy(lambda s: s.dependencies):
+        manifest = Lazy(lambda s: s.dependencies)
+    else:
+        manifest = False
```

Comment:
should be able to remove these and just call `self.dependencies` from the `skill_template`. This should also avoid it being triggered by the call from `upload.py`.